plone/protect/auto.py

```diff
@@ -107,6 +107,12 @@ def parseTree(self, result, encoding):
                                                    'compress',):
             return None
 
+        if isinstance(result, list):
+            # do not parse empty strings to omit warning log message
+            if len(result) == 1:
+                if result[0].strip() == u'':
```

Comment:
I would simplify this to `if not result[0].strip(): return None`.

Reply:
thanks @mauritsvanrees!
i was not sure what types of data structures can possibly be handed over to a transform's `transform` method.

if it is just either `repoze.xmliter.serializer.XMLSerializer` or a list containing one string, your simplification is fine.
if the list can possibly be empty we'd need to catch an IndexError

Reply:
With my simplification I only meant that one line: instead of `if X == u''` do `if not X`.

Reply:
Note: I am not quite sure what structures can end up in here either.